pinot-common/src/main/java/org/apache/pinot/common/utils/helix/HelixHelper.java

```diff
@@ -120,7 +127,6 @@ public Boolean call() {
             LOGGER.error("Caught permanent exception while updating ideal state for resource: {}", resourceName, e);
             throw e;
           } catch (Exception e) {
-            LOGGER.error("Caught exception while updating ideal state for resource: {}", resourceName, e);
```

Comment:
Suggest keeping this error log. It is unexpected

Reply:
good catch, I removed it accidentally